src/c++11/impl.cc

```diff
@@ -19,6 +19,9 @@ const char TraceKeyPrefix[] = "join:";
 const char TraceGUIDKey[] = "join:trace_guid";
```

Comment:
I'm assembling TraceJoinIds already, so I think this means I just need to remove the "join:" prefix. Correct?